test/parallel/test-fs-promises.js

```diff
@@ -133,6 +162,9 @@ function verifyStatObject(stat) {
     if (common.canCreateSymLink()) {
       const newLink = path.resolve(tmpDir, 'baz3.js');
       await symlink(newPath, newLink);
+      if (common.isOSX) {
+        await lchown(newLink, process.getuid(), process.getgid());
```

Comment:
If #21498 lands, this could become `!common.isWindows`.

Reply:
@cjihrig PTAL Thanks.